python/tk_multi_loader/action_manager.py

```diff
@@ -27,12 +27,11 @@ def __init__(self):
         """
         QtCore.QObject.__init__(self)
 
-    def get_actions_for_publish(self, sg_data, ui_area):
+    def get_actions_for_publishes(self, sg_data, ui_area):
```

Comment:
Other parts of the app, like the history view, won't support multiple selection for now for lack of a useful workflow so this is still required.